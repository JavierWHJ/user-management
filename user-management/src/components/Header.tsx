import React from 'react';
import { Navbar } from 'react-bootstrap';

const Header = () => {
    return (
        <Navbar bg="dark" variant="dark">
            <Navbar.Brand href="/">
            <img
                alt=""
                src="/logo.svg"
                width="30"
                height="30"
                className="d-inline-block align-top"
            />{' '}
            User Management
            </Navbar.Brand>
        </Navbar>
    );
}

export default Header;
