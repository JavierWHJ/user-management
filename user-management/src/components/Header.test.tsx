import React from "react";
import Header from "./Header";

import { render } from "@testing-library/react";

it("matches snapshot", () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});