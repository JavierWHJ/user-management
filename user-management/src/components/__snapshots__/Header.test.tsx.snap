// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="p-3 navbar navbar-expand navbar-dark bg-dark"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt=""
        class="d-inline-block align-top mr-2"
        height="30"
        src="/logo.png"
        width="40"
      />
       User Management
    </a>
  </nav>
</DocumentFragment>
`;
